Makefile

```diff
@@ -5,90 +5,53 @@ PYTHONPATH := $(shell which python 2>/dev/null || which python3 2>/dev/null)
 
 SHELL=bash
 
-# Get absolute path of venv directory
-BASE_DIR := $(shell realpath .)
-VENV=$(BASE_DIR)/.venv
-ifeq ($(OS),Windows_NT)
-    VENV_BIN := $(VENV)/Scripts
-else
-    VENV_BIN := $(VENV)/bin
-endif
-
-# Define the path to main Python pyproject.toml
-PYPROJECT_PATH := python/quantum-pecos/pyproject.toml
-
 # Virtual environment
 # -------------------
 
 .PHONY: venv
-venv:  ## Build Python virtual environment
-	$(PYTHONPATH) -m venv $(VENV)
+venv:  ## Create virtual environment and install all dependencies
+	@echo "Installing uv..."
+	$(PYTHONPATH) -m pip install --upgrade uv
```

Comment:
If the `Makefile` is intended to do the `uv` install, which I recommend against, it may be better to use `pipx install uv`. My recommendation is to ask people to install `uv` systemwide and only provide `sync` step here.